.github/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
```

Comment:
why is this file included in the pr?

Reply:
This should not been included. Let me fix it